pages/index.tsx

```diff
@@ -1,8 +1,18 @@
-import Head from 'next/head'
-import Image from 'next/image'
-import styles from '../styles/Home.module.css'
+import Head from 'next/head';
+import Image from 'next/image';
+import { wrapper } from '@/src/store';
+import serverApi from '../src/services/serverApi';
+import styles from '../styles/Home.module.css';
 
-export default function Home() {
+type PropsType = {
+  pageProps: {
+    members: {
+      [key: string]: string;
+    }
+  }
+}
```

Comment:
just a doubt here, so according to this type, every field in the members object is a value which is a string right?